app/src/main/java/com/antosik/benchproject/app/more/view/MoreFragmentNavigator.kt

```diff
@@ -0,0 +1,15 @@
+package com.antosik.benchproject.app.more.view
+
+import android.content.Intent
+import androidx.core.content.ContextCompat.startActivity
+import com.antosik.benchproject.app.common.navigator.FragmentNavigator
+import com.google.android.gms.oss.licenses.OssLicensesMenuActivity
+import dagger.hilt.android.scopes.ActivityRetainedScoped
+import javax.inject.Inject
+
+@ActivityRetainedScoped
+class MoreFragmentNavigator @Inject constructor() : FragmentNavigator() {
+
+    fun navigateToOssLicensesMenuActivity() =
+        startActivity(fragment!!.requireContext(), Intent(fragment!!.activity, OssLicensesMenuActivity::class.java), null)
```

Comment:
You could expose and use`requireFragment()` instead of `fragment!!`

Reply:
Also `fragment!!` is called two times so think about using some scope function here

Reply:
I changed it